src/models/euHealthCert/createEuHealthCert/createEuTestCert.ts

```diff
@@ -22,55 +22,61 @@ export const createEuTestCert = (
 
   const testHealthCerts: EuHealthCert[] = [];
   let incrementTestNumber = 1;
-  testData.forEach((item) => {
+  parsedFhirBundle.observations.forEach((item) => {
+    parsedFhirBundle.patient.fullName;
     const euName: EuNameParams = {
-      fnt: item.patientName.replace(/ /g, "<").toUpperCase(),
+      fnt: parsedFhirBundle.patient.fullName.replace(/ /g, "<").toUpperCase(),
     };
-    const dob = item.birthDate?.split("/")?.reverse()?.join("-");
+    const dob = parsedFhirBundle.patient.birthDate
+      .split("/")
+      ?.reverse()
+      ?.join("-");
     const meta: notarise.NotarisationMetadata = {
       reference,
       notarisedOn: dateString,
-      passportNumber: item.passportNumber,
+      passportNumber: parsedFhirBundle.patient.passportNumber,
       url: storedUrl,
     };
     // Set increment dose number + reference
     const uniqueRef = `${incrementTestNumber}${reference.toUpperCase()}`;
     // Set Unique Cert Id with prefix + version + country + unique ref
     const UniqueCertificateId = `URN:UVCI:01:SG:${uniqueRef}`;
     const euDccTestResultCode = Object.values(EUDccTestResult).includes(
-      item.testResultCode
+      item.observation.result.code as string
     )
-      ? item.testResultCode
-      : _.get(EUDccTestResult, item.testResultCode, null);
+      ? item.observation.result.code
+      : _.get(EUDccTestResult, item.observation.result.code as string, null);
 
     const testGroup: EuTestParams = {
       tg: "840539006",
       tt: "LP6464-4", // need to confirm with MOH for for Serology, it can either be [Nucleic acid amplification with probe detection] or [Rapid immunoassay]
       sc: moment
-        .tz(item.swabCollectionDate, "M/D/YY h:mm:ss A", "Asia/Singapore")
+        .tz(item.specimen.collectionDateTime, "Asia/Singapore")
```

Comment:
Do we still need to specify `"M/D/YY h:mm:ss A"`?

Oh and `moment` library is used here too haha

Reply:
ah.. i think this one and above one is related. i think i can removed both . one is convert from iso to M/D/YY h:mm:ss A and another one is convert to M/D/YY h:mm:ss A to iso :-D

Reply:
Ooh, btw does the EU QR `sc` field use `M/D/YY h:mm:ss` or ISO format?

Reply:
it's use iso format. this moment function helped to convert M/D/YY h:mm:ss to ios `format()`.